### DISCUSSION TITLE
Why the method burn is deprecated and now is disable?

### DISCUSSION THREAD

**Discussion by spielcrypto** [aptos-labs/aptos-developer-discussions#485] [Questions]
### Discord user ID

spielcrypto

### Describe your question in detail.

I have an application where users can burn scam nfts, now since burn method from object is deprecated, the applications just stopped to work because everyone receive a message from wallet `objects cannot be burnt `.

Can you please show me an alternative with **code** how can make disappear a NFT from an account with `move`?

Thanks in advance

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by gregnazario** [comment]
If they're not soulbound, you should be able to send them to a burn address such as `0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` to get rid of them.

If they are soulbound, your wallet should provide a way to hide them.

The `burn` function didn't actually burn the NFTs, but just change ownership of them in a way that caused some issues elsewhere.  See [AIP-99](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-99.md) for more details.

**Reply by spielcrypto** [reply]
To transfer to 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff or other address I need that the object allow ungate transfer which the scammer won't enable,  I built an app for community in order to burn NFTS and scammers Tokens and you are telling me that what I did was completely useless thanks for this update. What is not possible to do is developing a Blockchain based on just occurrences because tell me, why I have to continue developing on Aptos Blockchain if one day I build something with some functionalities provided by Aptos and another day your team will remove them breaking my app or making it useless like this time?

**Reply by Alexzander-Stone** [reply]
Hey there @dancespiele,

Due to the complexity behind burning objects and potential side-effects, developers decided to disable the usage of `burn`. Like Greg mentioned, you can check out AIP-99 for more information ([discussion](https://github.com/aptos-foundation/AIPs/issues/503), [governance voting that included AIP-99](https://governance.aptosfoundation.org/proposal/110)).

If you think we've made a mistake on this change, I recommend bringing up your concern in the above discussion post. Always great to have more developers contributing to the planned evolution of Aptos. 🫡

**Reply by gregnazario** [reply]
Let's chat and come up with a pollution that effectively hides the NFTs from wallets and all indexers, and we can use it to replace this functionality.  I didn't do enough reach out on this AIP, and it was my mistake for not providing an alternative. Dm me on telegram @greg_nazario

Been very busy and haven't had the chance to come back to it.

**Reply by spielcrypto** [reply]
I tried to find you on tg but for some reason I cannot see you

**Reply by spielcrypto** [reply]
I can find writing like this @gregnazario in tg, is that correct? or is someone pretending to be you?

**Comment by spielcrypto** [comment]
@gregnazario @Alexzander-Stone I commented in the discussion post https://github.com/aptos-foundation/AIPs/issues/503

**Comment by spielcrypto** [comment]
@gregnazario @Alexzander-Stone I didn't get any answer in the [post discussion](https://github.com/aptos-foundation/AIPs/issues/503) yet
